blocks/init/src/blocks/custom/carousel/docs/story.js

```diff
@@ -16,6 +16,6 @@ export default {
 };
 
 export const block = () => (
-  <Gutenberg props={blockDetails(manifest, globalManifest, item())} />
+  <Gutenberg props={blockDetails(manifest, globalManifest, [item(), item2(), item3()], 3)} />
```

Comment:
what does the parameter 3 at the end mean? Is the number of items inside the array?

Reply:
It is the number of times this items array will be duplicated.
This way you will have an output of

item1, item2, item3, item1, item2, item3, item1, item2, item3